ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -2557,6 +2629,8 @@ static vk_device ggml_vk_get_device(size_t idx) {
             device->coopmat_support = false;
         }
 
+        device->integer_dot_product = device->integer_dot_product && shader_integer_dot_product_props.integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated;
```

Comment:
```suggestion
        device->integer_dot_product = device->integer_dot_product && shader_integer_dot_product_props.integerDotProduct4x8BitPackedSignedAccelerated;
```

Reply:
I forgot about this, fixed. Funnily enough, in the other check for int dot I already used the non-saturating one. Oops.